stories/Atom/Buttons-and-labels/Chips/chips.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './chips.scss';
+
+export const Chips = ({ label }) => {
+  return (
+    <a class="chips chips-primary"
+      href="#"
+    >
+      { label }
+    </a>
+  );
+};
```

Comment:
@pallavikadam-srijan Please use bem for naming of classes

Reply:
@pallavikadam-srijan Can we remove the extra and give the chips class on the anchor itself